src/libraries/System.Runtime/tests/System.Threading.Tasks.Tests/System.Threading.Tasks.Tests.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetOS)' == 'browser'">
     <_WasmPThreadPoolUnusedSize>10</_WasmPThreadPoolUnusedSize>
+    <XunitShowProgress>true</XunitShowProgress>
```

Comment:
I filled https://github.com/dotnet/runtime/issues/104418 I hope we can pinpoint more specific root cause with your `XunitShowProgress` here